modules/Kconfig.mbedtls

```diff
@@ -29,8 +38,15 @@ config MBEDTLS_LIBRARY
 
 endchoice
 
+config CUSTOM_MBEDTLS_CFG_FILE
+	bool "Custom mbed TLS configuration file"
+	help
+	  Allow user defined input for the MBEDTLS_CFG_FILE setting.
+	  You can specify the actual configuration file using in
```

Comment:
>  You can specify the actual configuration file using in
	  MBEDTLS_CFG_FILE setting.

Apparently, a typo. Should be "You can specify the actual configuration file using **the** MBEDTLS_CFG_FILE setting."

Reply:
Thanks for spotting. Fixed.